src/main/kotlin/net/dungeonhub/application/connection/FlaggingConnection.kt

```
@@ -113,46 +130,30 @@ object FlaggingConnection : ExternalConnection {
-                    it
-                )!!
-            )
+    suspend fun isSafetyFlagged(uuid: UUID): FlagDetail? = try {
+        client.executeRawRequest {
+            url(Url("${ConfigProperty.SAFETY_API_URL}check/$uuid"))
+            parameter("type", "uuid")
+            header("Authorization", "Key ${ConfigProperty.SAFETY_API_KEY.value!!}")
+        }?.body<HypixelSafetyData>()?.let {
+            fromSafetyResponse(it)
         }
+    } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)